runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -3280,7 +3280,7 @@ done:;
 		buildInternalNativeStackFrame(REGISTER_ARGS);
 		updateVMStruct(REGISTER_ARGS);
 		_currentThread->privateFlags2 |= J9_PRIVATE_FLAGS2_UNSAFE_HANDLE_SIGBUS;
-		VM_UnsafeAPI::putBoolean(_currentThread, &_objectAccessBarrier, obj, offset, isVolatile, value);
+		VM_UnsafeAPI::putBoolean(_currentThread, &_objectAccessBarrier, obj, offset, isVolatile, value & 1);
```

Comment:
Shouldn't this be `(0 != value)` rather than `value & 1`?

Reply:
The spec says `the int value is narrowed by taking the bitwise AND of value and 1, resulting in value`.   
  
I don't see the difference between either approach since C++ bools will be 0 or 1.

However, the current implementation is consistent with what we did earlier in the interpreter.

Reply:
Perhaps this change should be made in UnsafeAPI.hpp so all users benefit?

Reply:
I think there only is one caller, but putting the mask in putBoolean instead makes sense.